common/arrays.test.ts

```diff
@@ -0,0 +1,14 @@
+import { findLastIndex } from './arrays'
+
+const makeCallbackFn = (searchValue: number) => (value: number) => value === searchValue
+
+describe('common/arrays', () => {
+  test('findLastIndex', () => {
+    expect(findLastIndex([], makeCallbackFn(1))).toBe(-1)
```

Comment:
a useful test to have would be with an array that contains 2 of the same value, to ensure it's actually picking the last index and not just any matching one